cli/cmd/container_registry.go

```diff
@@ -63,12 +63,22 @@ func init() {
 func containerRegistriesToTable(containerRegistries []api.ContainerRegistryRaw) [][]string {
 	var out [][]string
 	for _, cadata := range containerRegistries {
+
+		// @afiune LINK-1361
+		var state string
+		switch cadata.ContainerRegistryType() {
+		case api.InlineScannerContainerRegistry, api.ProxyScannerContainerRegistry:
+			state = "Ok"
```

Comment:
nit.  Would this logic be better incorporated here:
https://github.com/lacework/go-sdk/blob/f46a8c8ec140c482e8b38230581540405bd53f94/api/cloud_accounts.go#L237

Reply:
I can't do it there but, I think I can create another `StateString()` function at the `ContainerRegistryRaw` struct instead.